### PR TITLE
ERMS-2917: Vorbereitung zur eRechnung

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

**Review by klodav — APPROVED** [review]

File: app/web-app/setup/RefdataValue.csv [hunk 1]
```diff
@@ -630,7 +630,7 @@ e.invoice.portal,OZG-RE,OZG-RE,OZG-RE,,
 e.invoice.portal,service-bw.de,service-bw.de,service-bw.de,,
 e.invoice.portal,ZRE,ZRE,ZRE,,
 e.invoice.portal,zERIKA,zERIKA,zERIKA,,
-e.invoice.portal,Zentraler Rechnungseingang,Zentraler Rechnungseingang,Zentraler Rechnungseingang,,
+e.invoice.portal,Eigener zentraler Rechnungseingang,Eigener zentraler Rechnungseingang,Eigener zentraler Rechnungseingang,,
```
Comment (croqueGrec09) [inline app/web-app/setup/RefdataValue.csv:633]: You create a new value instead of modifying the existent one. The key has to remain as-is!

**Review by croqueGrec09 — CHANGES_REQUESTED** [review]
review refdata csv

File: app/web-app/setup/RefdataValue.csv (hunk 1 above)
Comment (croqueGrec09) [reply app/web-app/setup/RefdataValue.csv:633]: Moe will delete the old value

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**